### PR TITLE
Fixes three problems of my own identification/invention

### PR DESCRIPTION
**Pull request by Incoming5643**
1. Fixes meaty ores spawning with invisible "snack" meats, instead equal chances of human and mutant human meats
2. Fixes easyattachment limbs remaining in your hand sometimes (storing organs in null is dumb)
3. Mirrors the fix of all ghost role sleepers being exclusively male from my other, less mergable pr so it doesn't die there in the event that fails to be a thing

### CODE REVIEW

**Comment by Core0verload** [comment]
>Mirrors the fix of all ghost role sleepers being exclusively male from my other, less mergable pr 

This is not a correct way to do the fixing.

**Comment by coiax** [comment]
Yes it is. PRs are rarely atomic, and tend to fix other things in the area. It's not like people tend to have problems fixing merge conflicts from their own code anyway.

**Comment by Incoming5643** [comment]
@Core0verload 

If that PR is merged the changes disappear from this PR because that code is now the same as master
If this PR is merged the changes disappear from that PR for the same reason

It's git magic

File: code/modules/awaymissions/corpse.dm [hunk 1]
```diff
@@ -21,6 +21,10 @@
 	density = 1
 	anchored = 1
 
+/obj/effect/mob_spawn/New()
```
Comment (KorPhaeron) [inline code/modules/awaymissions/corpse.dm]: Why didnt you put this with the rest of the New() stuff like 3 lines down.

Or in the spawn/equip proc

Comment (Incoming5643) [inline code/modules/awaymissions/corpse.dm]: I didn't see it, I usually assume New() follows immediately after the object.

Comment (KorPhaeron) [inline code/modules/awaymissions/corpse.dm]: Never assume anything with my 2012 era shitcode (or my modern shitcode for that matter)

**Comment by Core0verload** [comment]
It's not about git magic, it's about fixing what isn't a bug.

**Pull request merged**